src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,79 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1FilesObject = (fileObject) => {
+    const fileNames = [];
+    const apiV1FileObject = {};
+
+    Object.keys(fileObject).forEach((key) => {
+      const fileName = fileObject[key].s3Path.split('/').pop();
+      fileNames.push(fileName);
+
+      apiV1FileObject[fileName] = {
+        path: fileObject[key].s3Path,
+        size: fileObject[key].size,
+        valid: fileObject[key].valid,
+        name: fileName,
+        upload: {
+          status: fileObject[key].uploadStatus,
+        },
+      };
+    });
+    return { apiV1FileObject, fileNames };
+  };
+
+  const sampleTechnologyConvert = (technology) => {
+    switch (technology) {
+      case '10x': return '10X Chromium';
+      default: return technology;
```

Comment:
There's something wrong if the technology is not recognized, so we can just throw an error

Reply:
I think this wasn't resolved yet, it's about trhowing an error with ("technology not recognized")